### PR TITLE
[DOC] tutorial/concepts: grammar fix

### PR DESCRIPTION
**Pull request by wvandertoorn**
Small fix in task description of 'Exercise: Static polymorphism with alphabets II'.
Fixes  1.3.1 of https://github.com/seqan/seqan3/issues/1608

### CODE REVIEW

File: doc/tutorial/concepts/index.md [hunk 1]
```diff
@@ -196,8 +196,7 @@ Try calling `print` with a different type, e.g. `int` to make sure that it does.
 \endsolution
 
 \assignment{Exercise: Static polymorphism with alphabets II}
-Adapt your previous solution to handle nucleotides different from the rest. For nucleotides it should print,
-both, the value and its complement.
+Adapt your previous solution to handle nucleotides differently from the rest. For nucleotides it should print both the value and its complement.
```
Comment (eseiler) [inline doc/tutorial/concepts/index.md]:
```suggestion
Adapt your previous solution to handle nucleotides differently from the rest. For nucleotides, it should print both the value and its complement.
```

**Comment by smehringer** [comment]
@marehr I assigned you as second reviewer for this small PR, hope thats fine

File: doc/tutorial/concepts/index.md [hunk 2]
```diff
@@ -195,9 +195,8 @@ Try calling `print` with a different type, e.g. `int` to make sure that it does.
 \include doc/tutorial/concepts/overloading_solution1.cpp
 \endsolution
 
-\assignment{Assignment 2: Static polymorphism with alphabets II}
-Adapt your previous solution to handle nucleotides different from the rest. For nucleotides it should print,
-both, the value and its complement.
+\assignment{Assignment 2:: Static polymorphism with alphabets II}
```
Comment (marehr) [inline doc/tutorial/concepts/index.md]:
```suggestion
\assignment{Assignment 2: Static polymorphism with alphabets II}
```

As I see it, all other files only have one colon, or?

**Comment by wvandertoorn** [comment]
@marehr good catch, not sure how that got in there

**Pull request merged**